### DISCUSSION TITLE
Can we grab metrics from a kubernetes cluster in order to be displayed into the web UI ?

### DISCUSSION THREAD

**Discussion by bioinfornatics** [cockpit-project/cockpit#18902] [Q&A] [answered]
Dear,
As we have an hybrid infrastructure both of VM / Metal / kube ,  I would like to know if we could use cockpit as global UI for all kind of machine.

So, after some read I found nothing about kube, here is my question:
 -> Can we grab metrics from a kubernetes cluster in order to be displayed into the web UI ?

Thanks for your help best regards

**Comment by marusak** [comment, marked as answer]
> Can we grab metrics from a kubernetes cluster in order to be displayed into the web UI ?

Cockpit is meant for management of one host not as fleet management. There are other tools that can agreagate metrics from multiple sources like https://prometheus.io/ . Cockpit helps to export metrics from a host to prometheus
